### DISCUSSION TITLE
How do you rebuild the elastic agents after updating SSL cert?

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#11284] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

256G

### Storage for /nsm

100G

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I've put my real SSL cert in place and restarted the so-nginx docker container and that is working great.  Do I need to rebuild the elastic agents to include the updated cert? If so, how do you go about doing that?

I read the documentation for 2.3 (https://docs.securityonion.net/en/2.3/osquery.html#agents) but running 'salt-call state.apply fleet.event_gen-packages' doesn't seem to work anymore:

```
# salt-call state.apply fleet.event_gen-packages
local:
    Data failed to compile:
----------
    No matching sls found for 'fleet.event_gen-packages' in env 'base'
```

Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment, marked as answer]
You should not need to do this.
